Dockerfile

```diff
@@ -32,5 +30,7 @@ RUN TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1+PTX" TORCH_NVCC_FLAGS="-Xfatbin -compr
     CMAKE_PREFIX_PATH="$(dirname $(which conda))/../" \
     pip install -v .
 
+RUN git clone https://github.com/pytorch/vision.git && cd vision && pip install -v .
```

Comment:
the dockerfile installs the `master` pytorch, and probably deserves the `master` torchvision.